DSC/dsc.py

```diff
@@ -608,19 +608,37 @@ def install_dsc_packages():
     openssl_version = get_openssl_version()
     omi_package_path = omi_package_prefix + openssl_version
     dsc_package_path = dsc_package_prefix + openssl_version
+    compiler_mitigated_omi_flag = get_compiler_mitigated_omi_flag()
     waagent.AddExtensionEvent(name=ExtensionShortName, op='InstallInProgress', isSuccess=True,
                               message="Installing omipackage version: " + omi_package_path + "; dsc package version: " + dsc_package_path)
     if distro_category == DistroCategory.debian:
-        deb_install_pkg(omi_package_path + '.x64.deb', 'omi', omi_major_version, omi_minor_version, omi_build,
+        deb_install_pkg(omi_package_path + '.ulinux' + compiler_mitigated_omi_flag + '.x64.deb', 'omi', omi_major_version, omi_minor_version, omi_build,
                         omi_release, ' --force-confold --force-confdef --refuse-downgrade ')
         deb_install_pkg(dsc_package_path + '.x64.deb', 'dsc', dsc_major_version, dsc_minor_version, dsc_build,
                         dsc_release, '')
     elif distro_category == DistroCategory.redhat or distro_category == DistroCategory.suse:
-        rpm_install_pkg(omi_package_path + '.x64.rpm', 'omi', omi_major_version, omi_minor_version, omi_build,
+        rpm_install_pkg(omi_package_path + '.ulinux' + compiler_mitigated_omi_flag + '.x64.rpm', 'omi', omi_major_version, omi_minor_version, omi_build,
                         omi_release)
         rpm_install_pkg(dsc_package_path + '.x64.rpm', 'dsc', dsc_major_version, dsc_minor_version, dsc_build,
                         dsc_release)
 
+def get_compiler_mitigated_omi_flag():
+    vm_supported, vm_dist, vm_ver = check_supported_OS()
+
+    if is_compiler_mitigated_omi_supported(vm_dist.lower(), vm_ver.lower()):
```

Comment:
We have list of supported distros, we should remove the rhel6 and oracle 6 from there also.

Reply:
Fixed, we have updated **supported_dists** in check_supported_OS()